.github/workflows/static_checks.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           if [ "${{ github.event_name }}" == "pull_request" ]; then
-            files=$(gh pr diff ${{ github.event.pull_request.number }} --name-only)
+            files=$(gh pr view ${{ github.event.pull_request.number }} --json files --template '{{range .files}}{{tablerow .path}}{{end}}')
```

Comment:
This fetches metadata from the PR and that includes a list of files, then strips just the path part into the same format as otherwise